lib/ui_layer/widgets/search_tab/search_result.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+import 'package:studentmap/domain_layer/building_manager/building.dart';
+
+class SearchResult extends StatelessWidget {
+  const SearchResult(this.building, {super.key});
+
+  final Building building;
+
+  @override
+  Widget build(BuildContext context) => Expanded(
```

Comment:
We shouldn't need the Expanded widget here  - it causes a lot of errors because it should only be used within a Row, Column or Flex

Reply:
I've fixed this issue in [SOF-74](https://github.com/softengsquad/studentmap/pull/38).